src/routes/user.ts

```diff
@@ -93,7 +130,45 @@ userRouter.get("/:userId", ensureAuthUser, async (req, res, next) => {
 /** A page to list all tweets liked by a user */
 userRouter.get("/:userId/likes", ensureAuthUser, async (req, res, next) => {
   const {userId} = req.params;
+  const currentUserId = req.authentication?.currentUserId;
+
   const userTimeline = await getUserLikesTimeline(Number(userId));
+  const followedCount = await getUserFollowedCount(Number(userId));
+  const followingCount = await getUserFollowingCount(Number(userId));
+
+  // ユーザー詳細ページのユーザーとの関係
+  const hasFollow = await hasUserFollow(Number(userId), Number(currentUserId));
+  // モーダルやタブの状況によって必要なときだけ取得するようにすると良さそう
+  const followers = await getFollowedUsers(Number(userId));
+  const followersWithHasFollow = [];
+  for (const follower of followers) {
+    const followerWithHasFollow = {
+      following: {
+        ...follower.following,
+        hasFollow: await hasUserFollow(
+          Number(follower.following.id),
+          Number(currentUserId)
+        ),
+      },
+    };
+    followersWithHasFollow.push(followerWithHasFollow);
+  }
```

Comment:
optional: こういう感じでタブが開いているときだけ実行されるようにしてもいいかも

```suggestion
  const followersWithHasFollow = [];
  if(req.query.activeTab === 'followers') {
      const followers = await getFollowedUsers(Number(userId));
      for (const follower of followers) {
        const followerWithHasFollow = {
          following: {
            ...follower.following,
            hasFollow: await hasUserFollow(
              Number(follower.following.id),
              Number(currentUserId)
            ),
          },
        };
        followersWithHasFollow.push(followerWithHasFollow);
      }
  }

```